ODIN_II/SRC/odin_ii.cpp

```diff
@@ -607,6 +628,14 @@ void get_options(int argc, char** argv) {
     if (global_args.permissive.value()) {
         warning_message(PARSE_ARGS, unknown_location, "%s", "Permissive flag is ON. Undefined behaviour may occur\n");
     }
+
+    if (global_args.mix_multipliers) {
+        int bit_value = (1 << mix_hard_blocks::MULTIPLIERS);
+        configuration.mix_soft_and_hard_logic = configuration.mix_soft_and_hard_logic | bit_value;
```

Comment:
I don't know if we need this flag.
can we just check that configuration.mults_mixing_ratio is not NAN? 
default to `NAN` and first check isnan, then check that its within a valid range (>= 0.0 && <= 1.0), otherwise set to NAN.

Reply:
I've reworked initialization, this is less complex.